### PR TITLE
Log changes for v1.6.353.

### PR DESCRIPTION
**Pull request by jeschkies**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,30 @@
...
+The default is `undefined`.
+
+### Fixed Issues
+
+- [MARATHON-8089](https://jira.mesosphere.com/browse/MARATHON-8089), [MARATHON-8088](https://jira.mesosphere.com/browse/MARATHON-8088) Provide `restricted` and `unrestricted` GPU scheduling behavior (#6052)
+- [MARAHTON-8112](https://jira.mesosphere.com/browse/MARATHON-8112) Open the persistence store before taking a backup using CLI (#6055)
+- [MARATHON-7751](https://jira.mesosphere.com/browse/MARATHON-7751) Do not overscale pods with persistent volumes (#6038)
+- Fixe a typo in the metrics conversion code where the `end` field of the `/metrics` endpoint was not properly populated (#6035)
+- [MARATHON-8063](https://jira.mesosphere.com/browse/MARATHON-8063) Enhance App API to reflect resources for executor (#5986)
+- Add deployment failure reason to event. (#6011)
```
Comment (zen-dog) [inline changelog.md]: Mention Ken's Unreachable Documentation?

**Comment by alenkacz** [comment]
thanks for helping to release this! 👍

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+- Fastly improved documentation on Unreachable Strategy
```
Comment (zen-dog) [inline changelog.md]: You mean `Vastly`? ;)
And could you please add a link to the doc/PR?

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,31 @@
...
+  whether or not applications require them.
+
+The default is `undefined`.
+
+### Fixed Issues
+
+- [MARATHON-8089](https://jira.mesosphere.com/browse/MARATHON-8089), [MARATHON-8088](https://jira.mesosphere.com/browse/MARATHON-8088) Provide `restricted` and `unrestricted` GPU scheduling behavior (#6052)
+- [MARAHTON-8112](https://jira.mesosphere.com/browse/MARATHON-8112) Open the persistence store before taking a backup using CLI (#6055)
+- [MARATHON-7751](https://jira.mesosphere.com/browse/MARATHON-7751) Do not overscale pods with persistent volumes (#6038)
+- Fixe a typo in the metrics conversion code where the `end` field of the `/metrics` endpoint was not properly populated (#6035)
```
Comment (kensipe) [inline changelog.md]: `Fixe` -> `Fixed`

Comment (jeschkies) [reply changelog.md]: Ok, that is a funny typo.

**Pull request merged**